index.bs

```diff
@@ -2161,25 +2169,25 @@ This enumeration's values describe the [=[RP]=]'s requirements for [=resident cr
 
 <div dfn-type="enum-value" dfn-for="ResidentKeyRequirement">
     :   <dfn>forbidden</dfn>
-    ::  This value indicates the authenticator MUST only create a non-resident credential.
+    ::  This value indicates the authenticator MUST create a [=Non-Resident Credential=].
```

Comment:
We don't usually capitalize terms like this outside their definition, so I'd prefer `[=non-resident credential=]` and `[=resident credential=]` throughout.

Reply:
Thanks - done.